bedrock/base/templates/macros-protocol.html

```diff
@@ -144,7 +180,10 @@
 {%- endmacro %}
 
 
-{# Call Out (compact): https://protocol.mozilla.org/patterns/organisms/call-out.html#call-out-compact #}
+{#
+  Call Out (compact): https://protocol.mozilla.org/patterns/organisms/call-out.html#call-out-compact
+  see call-out for detailed documentation
```

Comment:
Are we missing a description for these params?

Reply:
Oh, yup. Good catch.